src/engine/rendering/gui/GuiManager.cpp

```diff
@@ -0,0 +1,59 @@
+#include "GuiManager.hpp"
+
+#include "components/GuiComponent.hpp"
+#include "core/Application.hpp"
+
+namespace lei3d 
+{
+	GuiManager* GuiManager::s_instance = new GuiManager();
+
+	GuiManager& GuiManager::Instance()
+	{
+		return *s_instance;
+	}
+
+	void GuiManager::Init()
+	{
+		s_instance->m_guiFontShader = new Shader("./data/shaders/gui.vert", "./data/shaders/guifont.frag");
```

Comment:
Same here. I'm pretty sure the default destructor won't delete these automatically.

Reply:
I don't think it will ever be deleted but I can add deleting it to the destructor anyway.